admin/views/class-yoast-input-select.php

```diff
@@ -0,0 +1,103 @@
+<?php
+/**
+ * @package WPSEO\Admin
+ */
+
+/**
+ * Class for generating a html select.
+ */
+class Yoast_Input_Select {
```

Comment:
Adding an `add_attribute` for `class` and possibly more attributes would provide a lot more flexibility.
Class shouldn't have to be a parameter to the constructor.